libpod/options.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 var (
-	nameRegex = regexp.MustCompile("[a-zA-Z0-9_-]+")
+	nameRegex = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9_.-]+$")
```

Comment:
Now we require at least two characters. Is this intentional?

Reply:
Matches Docker.
```
docker volume create d
Error response from daemon: create d: volume name is too short, names should be at least two alphanumeric characters
```

Reply:
Thanks for checking, @rhatdan.

Note that this would technically mean to release a Podman 2.0.0. Users with just one character will now fail.

Reply:
Rather then do that, I would just allow single letter volumes.  I don't see a reason to not allow them.

I think the following would work:

 	nameRegex = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9_.-]*$")

Reply:
The regex looks good :+1: If @mheon is d'accord, this would my preference.

Reply:
SGTM, updating

Reply:
Updated to allow single-character names

Reply:
Thanks @mheon !